docs/Versioning.md

```diff
@@ -24,18 +24,18 @@ Following table summarizes the relationship between the ONNX Runtime version and
 opset version implemented in that release. Please note the Backwards and Forward compatibility notes above.
```

Comment:
Since you're updating this file, could you help revise this to: "The following table summarizes the relationship between the ONNX Runtime version and the ONNX opset version implemented in that release. Please note the backward compatibility notes above."